src/main/java/uk/gov/hmcts/darts/common/entity/JudgeEntity.java

```diff
@@ -20,21 +18,16 @@
 @EqualsAndHashCode(callSuper = false)
```

Comment:
Do we need to specify the attribute `(callSuper = false)` - this might be a bit of a hangover when I was getting rid of a warning where we were using lombok data annotation and I was extending of `VersionedEntity`.

We recently changed to using `@Getter @Setter` instead because of some issue Tony knows about. Equals contracts in entities are notoriously tricky. :)